src/apps/Permissions/AssignPermissionPanel.js

```diff
@@ -14,6 +14,8 @@ const DEFAULT_STATE = {
   roleIndex: -1,
 }
 
+const ANY_ACCOUNT_ADDRESS = '0xFFfFfFffFFfffFFfFFfFFFFFffFFFffffFfFFFfF'
```

Comment:
We should import this address from here: https://github.com/aragon/aragon/blob/65484affe61f152b6cd19b1a301a5edbb1187228/src/permissions.js#L9